interfaces/apparmor/template.go

```diff
@@ -490,7 +490,7 @@ var classicTemplate = `
   /** pix,
 
   capability,
-  change_profile,
+  change_profile unsafe /**,
```

Comment:
Do you need a similar change in docker-support interface?

Reply:
Yes, but it needs special handling that will be done in a separate PR.

Reply:
FYI, I have a plan for docker-support that, while not ideal, will work. That will be in a separate PR.